src/Sentry.AspNet/Internal/SystemWebVersionLocator.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Reflection;
+using System.Web;
+using Sentry.Internal;
+
+namespace Sentry.AspNet.Internal
+{
+    internal static class SystemWebVersionLocator
+    {
+        public static string? GetCurrent() => GetCurrent(ReleaseLocator.GetCurrent());
+        internal static string? GetCurrent(string? release)
+        {
+            if (release != null)
+            {
+                return release;
+            }
+            else if (HttpContext.Current?.ApplicationInstance?.GetType() is Type type)
```

Comment:
You just want to check if it's not null and assign it to a variable, right?

```suggestion 
            else if (HttpContext.Current?.ApplicationInstance?.GetType() is { } type)
```

Reply:
Correct